doc/deployment_examples/sinfonier/feeding_sinfonier.md

```diff
@@ -0,0 +1,408 @@
+  ```
+  {
+      "subscribeResponse": {
+          "duration": "P1M",
+          "subscriptionId": "your_subscription_id",
+          "throttling": "PT5S"
+      }
+  }  
+  ```
+  The value `subscriptionId` is very important. It's highly recommended to save it in case of you need to unsubscribe to the entity in the future.
+
+  If you receive another answer from `Orion` check your curl: `JSON Parse Error` is a typical error. In that case you will receive this message:
+  ```
+  {
+      "subscribeError": {
+          "errorCode": {
+              "code": "400",
+              "details": "JSON Parse Error",
+              "reasonPhrase": "Bad Request"
+          }
+      }
+  }   
+  ```
+
+3. Append some values in entity `Book1`.
+  ```
+  (curl localhost:1026/v1/updateContext -s -S --header 'Content-Type: application/json' --header 'Accept: application/json' -d @- | python -mjson.tool) <<EOF
+  {  
+      "contextElements": [
+          {
+              "type": "Book",
+              "isPattern": "false",
+              "id": "Book1",
+              "attributes": [
+                  {
+                      "name": "title",
+                      "type": "text",
+                      "value": "Game of Thrones"
+                  },
+                  {
+                      "name": "pages",
+                      "type": "integer",
+                      "value": "927"
+                  },
+                  {
+                      "name": "price",
+                      "type": "float",
+                      "value": "18.50"
+                  }
+              ]
+          }
+      ],
+      "updateAction": "APPEND"
+  }
+  EOF
+  ```
+  This action only store a value in the attributes, isn't a change for notify to `Cygnus`. You will receive:
+  ```
+  {
+      "contextResponses": [
+          {
+              "contextElement": {
+                  "attributes": [
+                      {
+                          "name": "title",
+                          "type": "text",
+                          "value": ""
+                      },
+                      {
+                          "name": "pages",
+                          "type": "integer",
+                          "value": ""
+                      },
+                      {
+                          "name": "price",
+                          "type": "float",
+                          "value": ""
+                      }
+                  ],
+                  "id": "Book1",
+                  "isPattern": "false",
+                  "type": "Book"
+              },
+              "statusCode": {
+                  "code": "200",
+                  "reasonPhrase": "OK"
+              }
+          }
+      ]
+  }   
+  ```
+
+4.  When you have your subscription and some appended values it's time to update them. This updates are going to be sent to Cygnus. The way to update is similiar to `APPEND` but you need to write `UPDATE` in curl. Let's see:
```

Comment:
The way for updating is through the `updateContext` operation, but using the `UPDATE` option.